src/NameApp.java

```diff
@@ -0,0 +1,313 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+/**
+ * Created by trevor.campbell on 4/29/16.
+ */
+public class NameApp {
+
+    public static void main(String[] args) {
+        String[] fileData = new String[4429];
```

Comment:
`4429` in this case is what we call a 'Magic Number'.  We don't know what it is (which is why it's magic). You generally deal with this by creating a constant, like `const int MAX_NUMBER_OF_NAMES = 4429` or something like that.